### DISCUSSION TITLE
Endpoint can not be displayed

### DISCUSSION THREAD

**Discussion by Evan-geng** [apache/skywalking#11505] [Q&A]
hello,everyboby.
  when i use skywalking 9.6.0, and java agent 8.16.0.i want to see endpoint of one service .it is blank and notice "More than 15,000 'grammar' tokens have been presented. To prevent Denial Of Service attacks, parsing has been cancelled.".i want to know how to resolve it fastly.tks

**Comment by Evan-geng** [comment]
![Uploading image.png…]()

**Reply by wu-sheng** [reply]
We can't see anything. You didn't upload anything successfully.

**Comment by wu-sheng** [comment]
> "More than 15,000 'grammar' tokens have been presented

This is not from SkyWalking. Maybe your own firewall or something?

**Reply by Evan-geng** [reply]
I previously saw someone raise the same issue, which was from an early version, but I still encountered it in the latest version.

**Comment by Evan-geng** [comment]
![screenshot-20231106-154749](https://github.com/apache/skywalking/assets/140053373/ffe5fe72-1fbb-4047-8d2f-99bcdf0b2ccf)

**Reply by wu-sheng** [reply]
Check the OAP logs, to see which component is complaining the 15k tokens. I didn't see this before.

**Reply by shuimuziyang** [reply]
I'm encountering the same issue as well

![image](https://github.com/apache/skywalking/assets/14255681/fca1b0ae-60b6-4989-94be-c5fb2a45a844)

**Comment by Superskyyy** [comment]
Looks like its from graphql, https://stackoverflow.com/questions/70933025/more-than-15000-parse-tokens-have-been-presented
